plugin/src/main/java/com/playmonumenta/epicstructures/managers/SpawnerBreakTrigger.java

```diff
@@ -76,6 +74,13 @@ public void spawnerBreakEvent(RespawningStructure structure) {
 				}
 			}
 		}
+		if (getSpawnerRatio() <= 0.2 && !structure.isConquered()) {
+			structure.conquerStructure();
+		}
```

Comment:
This actually shouldn't be 0.2. The structures are all set so their spawner count is the number to break to conquer. The actual number of spawners contained within the POI is higher than that. So same logic, just compare against 0, not 0.2.